### ISSUE TITLE
blank radio value as default selection

### ISSUE THREAD

**Issue by metasyn** [digitallinguistics/introdemo#30]
If the default selection is set to the value of noun, and someone accidentally forgets to change the value, it would be better for the POS value to be empty, than (potentially incorrectly) noun.

this pull requests adds

````
      <select class=wordPOS selected=''>
        <option value=''>p.o.s.</option>
```

**Comment by metasyn** [comment]
<select class=wordPOS selected=''>
        <option value=''>part of speech</option>

**Comment by metasyn** [comment]
Why was I calling this a radio? I don't know.

**Comment by amundo** [comment]
We need to decide about how to hook up event listeners to these views — one question is whether to use multiple WordViews as well as a PhraseView, or to have the PhraseView have events from the words be delegated up to the PhraseView. 

I think either way could work. 

Generally, we're moving into MVC territory already, where we're needing to keep the DOM in sync with data objects. So we'll have to decide on an approach to "data events."

**Comment by metasyn** [comment]
well, we would want the default value to `''` over `n.` either way, no ?

**Comment by amundo** [comment]
Yeah, I'd imagine so. For the current project we can just hardcore it as a value as you suggest.

**Comment by amundo** [comment]
By hardcore it I meant hardcode it. 

But hardcore it anyway.

**Comment by metasyn** [comment]
#hardcorecommit
